packages/mobile/android/app/src/main/java/org/celo/mobile/MainActivity.java

```diff
@@ -62,9 +62,7 @@ public void onPause() {
 
   @Override
   public void onReactContextInitialized(ReactContext context) {
-    context
-      .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
-      .emit("AppStartedLoading", "" + appStartedMillis);
+    // pass
```

Comment:
Nit: I think it could be helpful to write a little more in the comment to describe the historical context + why it's empty

Reply:
Actually just realized that we can remove the listener for this since it's unused with this change. Looking at it I also realized that we subscribed and unsubscribed in onPause / onResume instead of onCreate / onDestroy which means we would probably miss some events in the rare case of screen going dark. I don't think that was a huge amount but seems better not having to rely on that.